packages/react-tabster/src/hooks/useArrowNavigationGroup.ts

```diff
@@ -17,6 +18,10 @@ export interface UseArrowNavigationGroupOptions {
  * @param options - Options to configure keyboard navigation
  */
 export const useArrowNavigationGroup = (options?: UseArrowNavigationGroupOptions) => {
+  // A tabster instance is not necessary to generate tabster attributes
+  // but calling the hook will ensure that a tabster instance exists internally and avoids consumers doing the same
+  useTabster();
```

Comment:
The same thing happens in `useAccordion`, it's not awesome 😪

I have a proposal, can we wrap `getTabsterAttribute()` into `useTabsterAttributes()` hook and call `useTabster()` there. In this case we will be sure that Tabster has instance and can avoid direct calls. WDYT?

Reply:
Yup makes sense, also removed the reexport of `getTabsterAttribute`